larray/core.py

```diff
@@ -3873,10 +3894,20 @@ def __init__(self,
                 raise ValueError("length of axes %s does not match "
                                  "data shape %s" % (axes.shape, data.shape))
 
+        _attrs = _AttributeDict()
+        title = kwargs.pop('title', '')
```

Comment:
I'm still not sure what to do. For me, we should either consider all kwargs as attributes or force users to pass a dictionary/use a.attrs.new_key = new_value.